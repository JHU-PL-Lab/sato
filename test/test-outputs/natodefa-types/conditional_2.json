[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "f" ],
          "err_match_val": "fun x -> x + 1",
          "err_match_expected": "Boolean",
          "err_match_actual": "Function"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "if f then true else false",
    "err_steps": 58
  }
]
