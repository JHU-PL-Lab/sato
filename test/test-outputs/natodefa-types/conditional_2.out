Type errors for:
- Input sequence : []
- Found at expr  : if f then true
else false
--------------------
* Value    : f = fun x -> x + 1
* Expected : Boolean
* Actual   : Function
1 errors found. No further control flows exist.
